modules/web/src/app/shared/components/label-form/template.html

```diff
@@ -70,14 +79,22 @@
           <input matInput
                  (keyup)="check(i)"
                  name="value"
+                 [matAutocomplete]="staticValues"
                  formControlName="value" />
+          <mat-autocomplete #staticValues="matAutocomplete"
+                            (closed)="check(i)">
+            <mat-option *ngFor="let value of getKeyValues(label.get('key').value)"
+                        [value]="value">
+              {{value}}
+            </mat-option>
+          </mat-autocomplete>
           <mat-error *ngIf="label.get('value').errors?.validLabelValuePattern"> {{valueName}} not allowed. </mat-error>
           <mat-error *ngIf="label.get('value').errors?.validLabelValueLength"> {{valueName}} is too long. </mat-error>
         </mat-form-field>
         <button mat-icon-button
                 type="button"
                 class="delete-button"
-                [disabled]="!isRemovable(i)"
+                [disabled]="!isRemovable(i) || isProtectedKey(label.get('key').value, i)"
```

Comment:
IMO we should move `isProtectedKey(label.get('key').value, i)` inside the `isRemovable` method.